Game/character.py

```diff
@@ -0,0 +1,77 @@
+class Archer(object):
+
+	def __init__(self, name, weapon, armor, hitpoints = 75):
+		self.name = name
+		self.weapon = weapon
+		self.armor = armor
+		self.type = "archer"
+		self.hitpoints = hitpoints
+
+	def pickUpWeapon(self, weapon):
```

Comment:
It would be better to create a generic pick_up function and pass in the weapon or armor.  But as PEP8 rule suggests; if you repetitively type something 3 times, then create a generic function for it. But in this case, it's just 2 functions so you're fine.